### PR TITLE
PR title very explicit

### PR DESCRIPTION
**Pull request by clausia**
# PR description even more explicit

## Changes
Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum

## Additions
Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum

## Deletions
Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum

## Tests

Explica todas las pruebas que hiciste

Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum

### CODE REVIEW

File: codigo.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+x = 1
+if x == 1:
```
Comment (jennrub) [inline codigo.py:2]: Esto es un comentario sobre la línea de código que pregunta por el valor de la variable x

File: creado.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# This program prints Hello, world!
+
+print('Hello, world!')
```
Comment (jennrub) [inline creado.py:3]: comentario dos

**Review by jennrub — APPROVED** [review]
Comentario a toda la revisión

File: codigo.py (hunk 1 above)
Comment (clausia) [reply codigo.py:2]: No entiendo a que te refieres con tu comentario, me podrías dar más contexto?

**Review by brunormzg — COMMENTED** [review]
Archivos revisados

**Comment by brunormzg** [comment]
Comentario

**Pull request merged**